backend/kernelCI_app/views/issueDetailsBuildsView.py

```diff
@@ -39,23 +42,31 @@ def _fetch_incidents(self, *, issue_id: str, version: int) -> Optional[Dict]:
             for build in builds
         ]
 
-    def get(
-        self, _request, issue_id: Optional[str], version: Optional[str]
-    ) -> JsonResponse:
+    @extend_schema(responses=IssuesBuildResponse)
+    def get(self, _request, issue_id: Optional[str], version: Optional[str]):
```

Comment:
add the return type here 
```suggestion
    def get(self, _request, issue_id: Optional[str], version: Optional[str]) -> Response:
```